content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -46,8 +46,9 @@ different Kubernetes components.
 | `APIListChunking` | `true` | Beta | 1.9 | |
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
-| `AttachVolumeLimit` | `true` | Alpha | 1.11 | 1.11 |
+| `AttachVolumeLimit` | `false` | Alpha | 1.11 | 1.11 |
```

Comment:
This was set to `true` in Alpha, which I think is incorrect, right?

Reply:
As a GA feature, this should move to the section _Feature gates for graduated or deprecated features_

Reply:
Fixed

Reply:
I think we move the alpha and beta rows also to the graduated section below too